ext/nnstreamer/tensor_decoder/tensordec-imagesegment.c

```diff
@@ -23,10 +24,26 @@
  * @see		https://github.com/nnstreamer/nnstreamer
  * @author  Jihoon Lee <ulla4571@gmail.com>
  *          niklasjang <niklasjang@gmail.com>
- * @bug		No known bugs except for NYI items
+ *          Dongju Chae <dongju.chae@samsung.com>
+ * @bug     No known bugs except for NYI items
  *
  * option1: Decoder mode of image segmentation
  *          Available : tflite-deeplab
+ *          Available : snpe-deeplab
+ *          Available : snpe-depth
+ *
+ * expected models
+ * - tflite-deeplab : deeplabv3_257_mv_gpu.tflite (designed for embedded devices)
+ * - snpe-deeplab   : deeplabv3_mnv2_pascal_train_aug.dlc (converted from a TF model)
+ * - snpe-depth     : any snpe models (.dlc) producing grayscale images
+ *
+ * expected input dims
+ * - tflite-deeplab : #labels x width x height (float32, label probability)
+ *                    (e.g., 21 x 257 x 257)
+ * - snpe-deeplab   : width x height (float32, label index)
+ *                    (e.g., 1 x 513 x 513)
```

Comment:
segmentation model has output of 513 x 513 x 1
please check

Reply:
Thanks, updated.